src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const renderedTodos: Todo[] = todosFromServer.map(todo => ({
```

Comment:
Consider using a more descriptive name for the variable. `renderedTodos` doesn't describe what the variable is used for. 🤔